src/main/client/src/components/@layout/Footer.tsx

```diff
@@ -1,13 +1,83 @@
 import * as React from 'react'
-import { Box, Container, useColorModeValue } from '@chakra-ui/react'
-import { BUGREPORT_URL, KIRDEV_URL } from '../../utils/configurations'
+import { Box, Flex, HStack, Icon, Image, Link, StackProps, Text, useColorModeValue, VStack, Wrap } from '@chakra-ui/react'
+import { BUGREPORT_URL, KIRDEV_ORANGE, KIRDEV_URL } from 'utils/configurations'
+import { socialPages } from 'content/socialPages'
+import { FaHeart } from 'react-icons/fa'
+import { Container } from './Container'
 
-type FooterProps = {}
+type impresszumWrapItemProps = {
+  display: {
+    base?: string
+    md: string
+  }
+  key: string
+}
+const ImpressumWrapItem: React.FC<impresszumWrapItemProps> = ({ display, key }) => {
+  return (
+    <FooterWrapItem key={key} display={display}>
+      <Text marginBottom={3} align="center" fontSize="xl">
+        <Link textColor={KIRDEV_ORANGE} href="/impresszum">
+          Impresszum
+        </Link>
+        {' | '}
+        <Link textColor={KIRDEV_ORANGE} isExternal href="https://www.youtube.com/watch?v=YLR6tqJDHEc">
```

Comment:
Annyira Rickroll-ra számítottam...
![1pcs5p](https://user-images.githubusercontent.com/25415958/151186323-8105eb8c-6801-40c4-9df1-491251a8252c.jpg)

Reply:
Get ready for Jacksonrole